méli-mélo/th-empathy/death.html

```diff
@@ -0,0 +1,123 @@
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Benefits and Programs</a></h4>
+					<p>Applying for benefits to which you may be entitled following a death.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Representing someone who died</a></h4>
+					<p>The legal, financial, estate planning, and other aspects of representing someone who has died.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Taxes and finances related to death</a></h4>
+					<p>Information on managing finances of an individual who died, including preparing and filing their taxes.
+ </p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Estates and wills</a></h4>
+					<p>Find information about estate planning and preparing a will, as well as what to do when there is no will.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Death-related documents</a></h4>
+					<p>Documents you may need to obtain following a death.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+
+				<div class="col-sm-12 col-md-6 mrgn-bttm-lg">
+					<div class="empathy cards h-100 full-width">
+					<h4><a href="#" class="stretched-link">Contacting services</a></h4>
+					<p>Contact information for various federal or provincial programs, departments, or agencies.</p>
+					<span class="fas fa-arrow-right" aria-hidden="true"></span>
+					</div>
+				</div>
+			</div>
+		</div>
+
+		<div class="col-xs-12 col-md-4 cards empathy quiz-content mrgn-bttm-lg">
+				<h2>Someone died, <br>what do I do?</h2>
+				<p class="mrgn-tp-lg">Answer a few simple questions for guidance on your situation.</p>
+				<p class="mrgn-tp-lg"><span class="fas fa-clock"></span> This will take less than a minute.</p>
+				<a href="#" class="mrgn-tp-lg btn btn-default btn-empathy btn-block btn-lg">Start questionnaire</a>
+
+				<img class="quiz-img mrgn-tp-lg center-block hidden-xs hidden-sm" src="images/lloj-direction-cloud.256.png" alt="Someone died, what do I do?">
```

Comment:
This seems like a decorative image. The alt attribute should be empty.
```suggestion
				<img class="quiz-img mrgn-tp-lg center-block hidden-xs hidden-sm" src="images/lloj-direction-cloud.256.png" alt="">
```

Reply:
Resolved in latest commit